python/lib/communication/dmod/communication/evaluation_request.py

```diff
@@ -3,9 +3,9 @@
 import json
```

Comment:
I strictly ported the functionality and for better or worse ignored the purpose these classes are serving. I would think you could answer that question, @christophertubbs? I did not grep the repo, but I would think your code is what would be using these.

Reply:
It might be best as you alluded to, to decide if we _can_ remove / merge these classes and create a small PR that introduces that work after this has been merged in.

Reply:
It would. The curiosity I have is whether or not all of these can be boiled down into:

```python
class EvaluationRequest(Message, abc.ABC):
    """
    A request to be forwarded to the evaluation service
    """

    event_type: typing.ClassVar[MessageEventType] = MessageEventType.EVALUATION_REQUEST
    """ :class:`MessageEventType`: the event type for this message implementation """

    action: str
    parameters: typing.Dict[str, typing.Any] = Field(
        description="Parameters for the action",
        alias="action_parameters"
    )
```

Some of this may no longer matter since the primary method of communicating with the evaluation service is _not_ through these messages. Think a ticket should be added?

Reply:
Yeah, @christophertubbs, I say go ahead and open another ticket for this.  It seems worthwhile change but best handled as its own item.

Reply:
I just opened #339